crawlers/mooncrawl/mooncrawl/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .reporter import reporter
```

Comment:
The reason that you aren't able to install dependencies is that you are importing `reporter` in `__init__.py`, but `setup.py` imports `mooncrawl.version`, which implicitly loads `mooncrawl/__init__.py` which causes `mooncrawl/reporter.py` to load, and `humbug` isn't installed in the environment when you run `setup.py`.

Quick way to solve this is to just hard code version in `setup.py`.

I'm not sure what is the right way to solve it.